lnbits/tasks.py

```diff
@@ -131,45 +132,38 @@ async def check_pending_payments():
     the backend and also to delete expired invoices. Incoming payments will be
     checked only once, outgoing pending payments will be checked regularly.
     """
-    outgoing = True
-    incoming = True
 
     while settings.lnbits_running:
```

Comment:
This loop is terminated if there is an SQL error (or any kind of exception raised).
```suggestion
    while settings.lnbits_running:
       try:
            _check_pending_payments()
       except Exception as exc:
           logger.warning(exc)
           
       await asyncio.sleep(60 * 30)  # every 30 minutes
```